ci/test/pipeline.template.yml

```diff
@@ -16,6 +16,7 @@ dag: true
 
 env:
   CI_BUILDER_SCCACHE: 1
+  CI_BUILD_WITH_BAZEL: 1
```

Comment:
Do we also want to set this by default for people building locally?

Reply:
Not yet, I want to hold off just a little longer to make sure the C toolchain is stable on developers machines. Probably in a week or so though!